TravelBug/src/Components/VideoCard/VideoCard.jsx

```diff
@@ -1,16 +1,22 @@
 import "./VideoCard.css";
-
+import { useNavigate } from "react-router-dom";
 const VideoCard = ({ vid }) => {
+  const navigate = useNavigate();
   const { title, category, img, creator, _id } = vid;
+  const toSingleVideoPage = () => {
+    navigate(`/singlevideo/${_id}`);
+  };
   return (
     <>
       <div className="video-card">
-        <div className="image-container">
+        <div onClick={() => toSingleVideoPage()} className="image-container">
           <img src={`https://i.ytimg.com/vi/${_id}/0.jpg`} className="image" />
         </div>
         <div>
           <div className="title-box">
-            <h3 className="vid-title">{title}</h3>
+            <h3 onClick={() => toSingleVideoPage()} className="vid-title">
```

Comment:
Same here. No need of the arrow function.

Reply:
Done